core/solver/batch_dispatch.hpp

```diff
@@ -201,22 +195,12 @@ enum class log_type { simple_convergence_completion };
  * GKO_BATCH_INSTANTIATE will be prepended to the batch solver template
  * parameters.
  */
-#define GKO_BATCH_INSTANTIATE_VARGS(macro, ...) \
```

Comment:
I can remove it for these, because I do not need the macro explicitly in the first place of these macro.
In value_type variants, we need macro as an argument explicitly to call `macro(...)`.
Thus, we need one version for `macro(...)` and another VARGS version for `macro(..., __VA_ARGS__)`